static/js/factories/posts.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import R from 'ramda';
+
+import type { Post } from '../flow/discussionTypes';
+
+export const makePost = (isURLPost: boolean = false): Post => ({
```

Comment:
Curious if you think using something like [casual](https://github.com/boo1ean/casual) would be useful here since we do a similar thing with `factoryboy` in python?